tests/test_client.py

```diff
@@ -417,27 +397,85 @@ def test_get_trades_specified_success(self):
         startingBefore = datetime.datetime.utcnow().isoformat()
         with requests_mock.mock() as rm:
             json_obj = tests.test_json.mock_get_trades_json
-            uri = 'https://api.dydx.exchange/v1/dex/trades' \
-                + '?pairs=' + ','.join(PAIRS) \
+            uri = 'https://api.dydx.exchange/v2/trades' \
+                + '?market=' + ','.join(MARKETS) \
                 + '&limit=' + str(limit) \
                 + '&startingBefore=' + startingBefore
             rm.get(uri, json=json_obj)
             result = client.get_trades(
-                pairs=PAIRS,
+                market=MARKETS,
                 limit=limit,
                 startingBefore=startingBefore
             )
             assert result == json_obj
 
-    # ------------ create_order ------------
+    # ------------ replace_order ------------
+
+    def test_replace_order_success(self):
+        client = Client(PRIVATE_KEY_1)
+        with requests_mock.mock() as rm:
+            json_obj = tests.test_json.mock_create_order_json
+            rm.post(
+                'https://api.dydx.exchange/v1/dex/orders/replace',
+                additional_matcher=_create_additional_matcher(
+                    client,
+                    cancelId=ORDER_HASH
+                ),
+                json=json_obj
+            )
+            result = client.replace_order(
+                makerMarket=1,
+                takerMarket=0,
+                makerAmount=2000,
+                takerAmount=1000,
+                cancelId=ORDER_HASH
+            )
+            assert result == json_obj
+    # # ------------ create_order ------------
```

Comment:
nit: missing empty line. Why was `test_replace_order_success` moved?

Reply:
this was when i was merging sam's commit, i can move it back to the bottom.

Reply:
still no line here